### PR TITLE
Use table name as prefix instead of t1 

### PR DESCRIPTION
**Pull request by dmolineus**
Use table name as prefix instead of t1 to avoid issues with custom queries (See #42).

### CODE REVIEW

File: src/Model/Multilingual.php [hunk 1]
```diff
@@ -226,8 +228,9 @@ protected static function applyOptionsToQueryBuilder(QueryBuilder $qb, array $op
                     $qb->andWhere($column);
                 }
             } else {
-                // Default is likely t1
-                $qb->andWhere('t1.'.$options['column'].'=?');
+                // Default is likely fallback table
+                $table = static::getTable();
+                $qb->andWhere("$table.{$options['column']}=?");
```
Comment (dmolineus) [inline src/Model/Multilingual.php]: Always appending the prefix to the column seems wrong to me as you never know if a user would set it for it own. I didn't change it for BC reasons.

Comment (Toflar) [reply src/Model/Multilingual.php]: Could we check for the presence of a dot? ;-)

Comment (Toflar) [reply src/Model/Multilingual.php]: https://github.com/contao/core-bundle/blob/master/src/Resources/contao/library/Contao/Model.php#L627 there are already such assuptions. I think it should be just fine to only prefix if there's no `.` in it for maximum BC.

**Comment by Toflar** [comment]
I cannot release this as a hotfix for version 3 becaue it would break existing extensions.
So I merge this into develop and it will be released as version 4. Having a look at the other issues now.

**Pull request merged**